### DISCUSSION TITLE
Window11 系统打不开蓝灯，显示localhost 已拒绝连接。8位号码是：74822341

### DISCUSSION THREAD

**Discussion by peteryang0802** [getlantern/lantern-forum-cn#729] [Q&A]
Window11 系统打不开蓝灯，显示localhost 已拒绝连接
显示： 
无法访问此网站localhost 拒绝了我们的连接请求。 请试试以下办法： 检查网络连接 检查代理服务器和防火墙 ERR_CONNECTION_REFUSE D

通过安卓版提交了支持，8位号码是：74822341

附上lantern.log 文件。

[lantern.log](https://github.com/user-attachments/files/16318607/lantern.log)

**Comment by MaxMKMax** [comment]
@peteryang0802 好的. 非常感谢您. 我会帮您提交给技术团队修复问题的.

**Comment by peteryang0802** [comment]
你好，算了，我卸载了。
麻烦把这个问题close吧。
我安装了preview版本，但是发现一个问题，提交了支持，号码是：74827570
问题是：preview软件，没法确认成PRO版本，或者说，一会儿显示是PRO，重启后又变成普通版了。
***@***.***
---- 回复的原邮件 ----
发件人Mike ***@***.***>发送日期2024年7月22日 ***@***.***>***@***.***>,
***@***.***>主题Re: [getlantern/lantern-forum-cn] Window11 系统打不开蓝灯，显示localhost 已拒绝连接。8位号码是：74822341 (Discussion #729)
@peteryang0802 好的. 非常感谢您. 我会帮您提交给技术团队修复问题的.

**Reply by fengg80** [reply]
我也是这个问题

**Comment by chenrenying** [comment]
我也是这个问题，困扰了我好久了。正式版打不开，preview版本无半授权为专业版，或者说是提示已经绑定，再实际上并没绑定成功，再登录就成免费版的了。

**Reply by ycjdixon** [reply]
他们技术团队不工作的，一星期开一次会，然后就没有然后了……

**Reply by MaxMKMax** [reply]
@chenrenying 我又一直在帮你们反馈的. 因为还有别的国家遇到问题, 所以需要排一下队给技术团队. 不好意思让你们久等了.

**Comment by peteryang0802** [comment]
这个问题我放弃了，我安装了preview的版本，没这个问题了。
但是preview版本有问题，一会儿是pro，一会儿又不是pro...
peteryang
***@***.***
---- 回复的原邮件 ----
***@***.***>发送日期2024年7月23日 ***@***.***>***@***.***>,
***@***.***>主题Re: [getlantern/lantern-forum-cn] Window11 系统打不开蓝灯，显示localhost 已拒绝连接。8位号码是：74822341 (Discussion #729)
我也是这个问题
